packages/content/src/portal.ts

```diff
@@ -117,11 +118,16 @@ export function itemToContent(item: IItem): IHubContent {
 }
 
 /**
+ * DEPRECATED: Use getFamily() instead.
+ *
  * get the HubType for a given item or item type
  *
  * @param itemOrType an item or item.type
  */
 export function getItemHubType(itemOrType: IItem | string): HubType {
+  console.warn(
```

Comment:
Any reason to not use `Logger.warn`?

Reply:
Like all the other deprecations, this is temporary code, so I think `console.log()` is fine, even better that `Logger` in cases where that would have to be added to the `import`s.